_posts/2022-11-07-road-to-quarkus-3.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: 'Starting Quarkus 3'
+date: 2022-11-07
+tags: announcement
+synopsis: 'Quarkus 3 is coming with new features and a move to Jakarta named APIs.'
+author: maxandersen
+---
+
+Quarkus 3 will continue the tradition of propelling Java development forward by moving fast and bring a cutting-edge user experience with the latest technologies.
+
+Happy to have a first Quarkus 3 Alpha available for users to try out and extension writers to integrate with to get early feedback.
+
+## What is in it?
+
+The current planned list of highlights for Quarkus 3 are:
+
+  - *Hibernate 6* - faster, safer, better (Check out https://www.youtube.com/watch?v=pc6QIwx0EL0[this presentation] to see how awesome it is).
+  - *Jakarta EE 10* API and packages - following the latest evolution of the Java world
+  - *Micro profile 6* - the latest update of all the MP specifications continuing evolving alongside the state of the practice
+  - *HTTP/3* support (aka *Quic*) - the https://www.redhat.com/architect/http3[new HTTP version] fixing the head-of-line blocking problem, using UDP.
+  - *IO_URING* - the next generation of asynchronous IO support. This is a complete game changer if you are interested in performance and efficiency (response time, latency…)
+  - *Virtual threads* (aka Loom) and *Structured Concurrency Support* - based on the lessons learned from our https://www.redhat.com/architect/http3[initial integration], we are now looking into expanding and improving the support.
+  - *java.util.concurrent.Flow* - Time to evolve; Flow has been in the JDK since Java 9. Thus we want to move from Reactive Streams to java.util.concurrent.Flow.
+  - *Documentation* - Work started on better and more structured documentation following the https://diataxis.fr/[Diataxis] framework principles.
+  - *Tooling for updates* - going to be a new major version, we want to make the migration as smooth as possible. Thus we will provide migration tools.
+  - A new *gRPC server*, more flexible and easier to maintain
+  - A *revamped dev UI*
+  - <Your extension awesome feature/improvements here>
+
+## Release Plan
+
+We will continue the monthly cadence release of the 2.x version of Quarkus. Quarkus 3.x preview (alphas, betas) will be released in parallel. This blog marks the start of the first Quarkus 3 Alpha.
+
+It will take more than a month to integrate the change set we want and collect feedback (from users, and extension maintainers...) from the previous release. Thus, 3.x release cadence will be a bit slower than usual.
+
+Thus the intent is to have a time period of some months to allow continuous integration and start releasing a Quarkus Platform with the members that are ready to do and then continuously include more.
+
+Proposed plan:
+
+  - Start doing 3.0.0.Alpha core and platform releases at regular intervals to allow easy integration builds, starting first from 2.13.3 in the coming week
```

Comment:
So we will announce 3.0.0.Alpha1 separately? I think it kinda makes sense, just checking.
```suggestion
  - Start doing 3.0.0.Alpha core and platform releases at regular intervals to allow easy integration builds, starting first from 2.13.3 in the coming week
```

Reply:
Yeah, I was thinking of keeping 2.x and 3.x announce separate - possibly just link from one to the other if relevant.